src/runtime-rs/crates/resource/src/rootfs/nydus_rootfs.rs

```diff
@@ -143,4 +146,9 @@ impl Rootfs for NydusRootfs {
     async fn get_storage(&self) -> Option<Storage> {
         Some(self.rootfs.clone())
     }
+
+    async fn cleanup(&self) -> Result<()> {
+        warn!(sl!(), "Cleaning up Nydus Rootfs is still unimplemented.");
```

Comment:
You can add a `TODO` here, and raise an issue for this.

Reply:
Ok, it is not just the `NydusRootfs`, some of the implementations of share fs suffer from this problem as well.